testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java

```diff
@@ -689,4 +688,104 @@ public void verifyFT77()
                 expectedJsonPattern, ADMIN_USERNAME, ADMIN_PASSWORD));
     }
 
+    @Test
+    public void verifyFT76() throws IOException, InterruptedException {
+        /*
+         * To check that only message for Active Pack should be returned from
+         * inbox after 7 days of user's subscription gets completed for 72Weeks
+         * Pack while user is subscribed for both Pack.
+         */
+        setupData();
+
+        Subscriber mctsSubscriber = new Subscriber(9999911122L);
+        mctsSubscriber.setDateOfBirth(DateTime.now());
+        subscriberDataService.create(mctsSubscriber);
+
+        // create subscription to child pack for which start date is as per DOB
+        Subscription childPackSubscription = subscriptionService
+                .createSubscription(9999911122L, rh.hindiLanguage(),
+                        sh.childPack(),
+                        SubscriptionOrigin.MCTS_IMPORT);
+        mctsSubscriber = subscriberDataService.findByCallingNumber(9999911122L);
+
+        // due to subscription rules detailed in #157, we need to clear out the
+        // DOB and set an LMP in order to
+        // create a second subscription for this MCTS subscriber
+        mctsSubscriber.setDateOfBirth(null);
+        mctsSubscriber.setLastMenstrualPeriod(DateTime.now());
+        subscriberDataService.update(mctsSubscriber);
+
+        // create subscription to pregnancy pack
+        Subscription pregnancyPackSubscription = subscriptionService
+                .createSubscription(9999911122L, rh.hindiLanguage(),
+                        sh.pregnancyPack(),
+                        SubscriptionOrigin.MCTS_IMPORT);
+        // update pregnancy subscription pack to mark complete
+        // setting the subscription to have ended more than a week ago -- no
+        // message should be returned
+        subscriptionService.updateStartDate(pregnancyPackSubscription, DateTime
+                .now().minusDays(512 + 90));
+
+        String expectedJsonResponse = "{\"inboxSubscriptionDetailList\":[{\"subscriptionId\":\""
+                + childPackSubscription.getSubscriptionId()
+                + "\",\"subscriptionPack\":\"childPack\",\"inboxWeekId\":\"w1_1\",\"contentFileName\":\"w1_1.wav\"}]}";
+
+        HttpGet httpGet = createHttpGet(true, "9999911122", true,
+                "123456789012345");
+        assertTrue(SimpleHttpClient.execHttpRequest(httpGet, HttpStatus.SC_OK,
+                expectedJsonResponse, ADMIN_USERNAME, ADMIN_PASSWORD));
+    }
+
+    @Test
+    public void verifyFT80() throws IOException, InterruptedException {
+        /*
+         * To check that only message for Active Pack should be returned from
+         * inbox after 7 days of user's subscription gets completed for 48Weeks
+         * Pack while user is subscribed for both Pack.
+         */
+        setupData();
```

Comment:
setupData() is called automatically on line 113 before each test, it's not needed here.